homeassistant/components/soundtouch/media_player.py

```diff
@@ -350,6 +352,21 @@ def play_media(self, media_type, media_id, **kwargs):
             else:
                 _LOGGER.warning("Unable to find preset with id %s", media_id)
 
+    def select_source(self, source):
+        """
+        Select the source to play audio from.
+
+        :param source:source to select
+        """
+        if source == "AUX":
+            _LOGGER.info("Selecting source AUX")
```

Comment:
Use debug here to not spam the logs.
```suggestion
            _LOGGER.debug("Selecting source AUX")
```

Reply:
Thanks, changed.
I was more immitating the surrounding code than really thinking about it.